packages/entity-browser/src/components/DetailView/DetailForm.js

```diff
@@ -80,7 +84,19 @@ DetailForm.propTypes = {
   submitting: React.PropTypes.bool,
   submitFailed: React.PropTypes.bool,
   submitSucceeded: React.PropTypes.bool,
-  anyTouched: React.PropTypes.bool
+  anyTouched: React.PropTypes.bool,
+  loadRelationEntities: React.PropTypes.func,
+  stores: React.PropTypes.shape({
```

Comment:
name 'store' does not really explain what exactly is stored. Its only for relatedEntities, right? I would name id like that